src/rest/publish.ts

```diff
@@ -99,7 +99,8 @@ const publishRest = new Vue({
         throw error;
       }
     },
-    async assetPaths(identifier: string, version: string, location: string): Promise<string[]> {
+    // TODO: define return type here
+    async assetPaths(identifier: string, version: string, location: string): Promise<any> {
```

Comment:
```suggestion
    async assetPaths(identifier: string, version: string, location: string): Promise<{ folders: AssetFolder[], files: AssetFile[] }> {
```
I'm not sure it's worth it's own type, so using a custom type definition seems like the easy way out.
You probably need to import those types in this file though.